airstack-modules/modules/airstack/nft-marketplace/nft.ts

```diff
@@ -42,9 +42,9 @@ export namespace nft {
             let paymentToken = getOrCreateAirToken(chainID, NftSales[i].paymentToken.toHexString());
 
             // Account
-            let buyerAccount = handleAccountCreation(chainID, NftSales[i].buyer.toHexString(), block.id);
-            let sellerAccount = handleAccountCreation(chainID, NftSales[i].seller.toHexString(), block.id);
-            let feeAccount = handleAccountCreation(chainID, NftSales[i].protocolFeesBeneficiary.toHexString(), block.id);
+            let buyerAccount = getOrCreateAirAccount(chainID, NftSales[i].buyer.toHexString(), block);
```

Comment:
Where are you saving these accounts?

Reply:
fixing this, by saving inside the air account function

Reply:
done, rarible subgraph integration, testing and deployment in #43